cmd/simulator/txs/agent.go

```diff
@@ -53,6 +56,9 @@ func (a issueNAgent[T]) Execute(ctx context.Context) error {
 	}
 
 	txChan := a.sequence.Chan()
+	totalTxs := len(txChan)
```

Comment:
could we count the txs instead of taking the chan len?

Reply:
yup

Reply:
I need a totalCount to know when to fully stop

Reply:
see the comment below